src/prngs/chacha20.c

```diff
@@ -110,6 +111,7 @@ int chacha20_prng_ready(prng_state *prng)
 unsigned long chacha20_prng_read(unsigned char *out, unsigned long outlen, prng_state *prng)
 {
    LTC_ARGCHK(prng != NULL);
+   if (!prng->chacha.ready) return 0;
```

Comment:
I just asked myself if it'd make sense to move the `ready` flag to the `prng_state`?